test.js

```diff
@@ -37,19 +37,61 @@ test('input string date returns a week number', t => {
 	t.is(result, expectedResult);
 });
 
-test('yesterday is same or previous week', t => {
-	const resultYesterday = weekNumber.yesterday();
-	const resultToday = weekNumber();
+test('yesterday is previous week if current day is monday and not at the start of the year', t => {
+	// Mock current time to Monday (September 2, 2019)
+	const realNow = Date.now;
+	Date.now = () => new Date('September 2, 2019').getTime();
```

Comment:
One issue with mocking Date.now() is that it's global and could interfere with parallel tests. I've added test.serial() to those tests.

Reply:
Thanks, good catch 👍